server/eclipse-project/src/test/java/au/org/democracydevelopers/corla/endpoint/GetAssertionsTests.java

```diff
@@ -0,0 +1,248 @@
+import static org.testng.Assert.assertEquals;
+
+/**
+ * Test the GetAssertions endpoint, both CSV and JSON versions. The response is supposed to be a zip file containing
+ * the assertions.
+ * TODO This really isn't a completely comprehensive set of tests yet. We also need:
+ * - API testing
+ * - Testing for retrieving the data from the zip.
+ * - Testing that the service throws appropriate exceptions if the raire service connection isn't set up properly.
+ * See <a href="https://github.com/DemocracyDevelopers/colorado-rla/issues/125">...</a>
+ */
+public class GetAssertionsTests {
+
+    private static final Logger LOGGER = LogManager.getLogger(GetAssertionsTests.class);
+
+    private final static String boulderMayoral = "City of Boulder Mayoral Candidates";
+    private final static String tinyIRV = "TinyExample1";
+
+    private final Choice alice = new Choice("Alice", "", false, false);
+    private final Choice bob = new Choice("Bob", "", false, false);
+    private final Choice chuan = new Choice("Chuan", "", false, false);
+
+    private final List<Choice> boulderMayoralCandidates = List.of(
+            new Choice("Aaron Brockett", "", false, false),
+            new Choice("Nicole Speer", "", false, false),
+            new Choice("Bob Yates", "", false, false),
+            new Choice("Paul Tweedlie", "", false, false)
+    );
+
+    /**
+     * Two IRV Contests for mocking the IRVContestCollector.
+     */
+    private final Contest tinyIRVExample = new Contest(tinyIRV, new County("Arapahoe", 3L), ContestType.IRV.toString(),
+            List.of(alice, bob, chuan), 3, 1, 0);
+    private final ContestResult tinyIRVContestResult = new ContestResult(tinyIRV);
+    private final Contest boulderMayoralContest = new Contest(boulderMayoral, new County("Boulder", 7L), ContestType.IRV.toString(),
+            boulderMayoralCandidates, 4, 1, 0);
+    private final ContestResult boulderIRVContestResult = new ContestResult(boulderMayoral);
+    private final List<ContestResult> mockedIRVContestResults = List.of(boulderIRVContestResult, tinyIRVContestResult);
+
+
+    /**
+     * Wiremock server for mocking the raire service.
+     */
+    final WireMockServer wireMockRaireServer = new WireMockServer();
+
+    /**
+     * Initialise mocked objects prior to the first test.
+     */
+    @BeforeClass
+    public void initMocks() {
+        MockitoAnnotations.openMocks(this);
+
+        boulderIRVContestResult.setAuditReason(AuditReason.COUNTY_WIDE_CONTEST);
+        boulderIRVContestResult.setBallotCount(100000L);
+        boulderIRVContestResult.setWinners(Set.of("Aaron Brockett"));
+        boulderIRVContestResult.addContests(Set.of(boulderMayoralContest));
+
+        tinyIRVContestResult.setAuditReason(AuditReason.COUNTY_WIDE_CONTEST);
+        tinyIRVContestResult.setBallotCount(10L);
+        tinyIRVContestResult.setWinners(Set.of("Alice"));
+        tinyIRVContestResult.addContests(Set.of(tinyIRVExample));
+
```

Comment:
See earlier comment: the 200 should be a constant somewhere. Also, you've made constants elsewhere for the /raire/get-assertions etc but are hardcoding here.